api/xds_protocol.rst

```diff
@@ -613,6 +613,43 @@ adding/removing/updating clusters. On the other hand, routes are not
 warmed, i.e., the management plane must ensure that clusters referenced
 by a route are in place, before pushing the updates for a route.
 
+.. _xds_protocol_TTL:
+
+TTL
+~~~
+
+In the event that the management server becomes unreachable, the last known configuration received
+by Envoy will persist until the connection is reestablished. For some services, this may not be
+desirable. For example, in the case of a fault injection service, a management server crash at the
+wrong time may leave Envoy in an undesirable state. The TTL setting allows Envoy to remove a set of
```

Comment:
If we remove a resource that is required by the configuration, doesn't that mean the configuration is invalid?  It's not clear to me how removing a resource could simply disable a feature.

Reply:
The fault injection service is enabled by a particular configuration that is delivered through xDS. A fault injection test is disabled by the removal of that resource.

Reply:
I think this was answered by @mattklein123 below

Reply:
The fault injection feature is enabled by an xDS setting sent to the client in a resource. Removing that resource disables the fault injection feature.

Reply:
Yes, this question has been addressed.